src/pubkey.cpp

```diff
@@ -24,7 +24,7 @@ secp256k1_context* secp256k1_context_verify = nullptr;
  *  strict DER before being passed to this module, and we know it supports all
  *  violations present in the blockchain before that point.
  */
-static int ecdsa_signature_parse_der_lax(const secp256k1_context* ctx, secp256k1_ecdsa_signature* sig, const unsigned char *input, size_t inputlen) {
+int ecdsa_signature_parse_der_lax(const secp256k1_context* ctx, secp256k1_ecdsa_signature* sig, const unsigned char *input, size_t inputlen) {
```

Comment:
why can't you call `CPubKey::CheckLowS`, which directly calls this?

Reply:
I want to fuzz both in order to a.) maximize coverage, and b.) to avoid assuming anything about the interdependence of these two function (which may change over time).